networking/cardano-sl-networking.cabal

```diff
@@ -64,6 +64,7 @@ Library
                       , async
                       , attoparsec
                       , base
+                      , cardano-sl-util
```

Comment:
If `networking` now depends on `util` is probably worth updating the dependency info in the `stack.yaml` file.

Reply:
Dependency info in `stack.yaml`?

Reply:
There a comment in the `stack.yaml` file which currently looks like:
```
# networking → binary → util → crypto → core → db → [lrc, infra]
#              → [ssc, txp, update, delegation] → block → lib → ...
# Then we have two branches:
# ... → client → generator → [auxx, explorer, wallet] → wallet-new
# ... → [node, tools]
```
Your change makes `networking` depend on `util` so that coment should be updated.

Reply:
I'm tempted to just get rid of this comment altogether. Do you think it's of any good use? If I want to know dependencies I can check the cabal file of that thing.